src/drivers/drv_sensor.h

```diff
@@ -142,26 +142,27 @@
 #define DRV_DIST_DEVTYPE_TERARANGER   0x75
 #define DRV_DIST_DEVTYPE_VL53L0X      0x76
 #define DRV_POWER_DEVTYPE_INA226      0x77
-#define DRV_POWER_DEVTYPE_VOXLPM      0x78
+#define DRV_POWER_DEVTYPE_INA228      0x78
+#define DRV_POWER_DEVTYPE_VOXLPM      0x7a
 
-#define DRV_LED_DEVTYPE_RGBLED        0x7a
-#define DRV_LED_DEVTYPE_RGBLED_NCP5623C 0x7b
-#define DRV_BAT_DEVTYPE_SMBUS         0x7c
-#define DRV_SENS_DEVTYPE_IRLOCK       0x7d
-#define DRV_SENS_DEVTYPE_PCF8583      0x7e
-#define DRV_TEL_DEVTYPE_BST           0x7f
+#define DRV_LED_DEVTYPE_RGBLED        0x7b
+#define DRV_LED_DEVTYPE_RGBLED_NCP5623C 0x7c
+#define DRV_BAT_DEVTYPE_SMBUS         0x7d
+#define DRV_SENS_DEVTYPE_IRLOCK       0x7e
+#define DRV_SENS_DEVTYPE_PCF8583      0x7f
+#define DRV_TEL_DEVTYPE_BST           0x80
 
 // Generic types for unknown CAN sensors
-#define DRV_ACC_DEVTYPE_UAVCAN	0x80
-#define DRV_BARO_DEVTYPE_UAVCAN	0x81
-#define DRV_BAT_DEVTYPE_UAVCAN	0x82
-#define DRV_DIFF_PRESS_DEVTYPE_UAVCAN	0x83
-#define DRV_FLOW_DEVTYPE_UAVCAN	0x84
-#define DRV_GPS_DEVTYPE_UAVCAN	0x85
-#define DRV_GYR_DEVTYPE_UAVCAN	0x86
-#define DRV_IMU_DEVTYPE_UAVCAN	0x87
-#define DRV_MAG_DEVTYPE_UAVCAN	0x88
-#define DRV_DIST_DEVTYPE_UAVCAN	0x89
+#define DRV_ACC_DEVTYPE_UAVCAN	0x81
+#define DRV_BARO_DEVTYPE_UAVCAN	0x82
+#define DRV_BAT_DEVTYPE_UAVCAN	0x83
+#define DRV_DIFF_PRESS_DEVTYPE_UAVCAN	0x84
+#define DRV_FLOW_DEVTYPE_UAVCAN	0x85
+#define DRV_GPS_DEVTYPE_UAVCAN	0x86
+#define DRV_GYR_DEVTYPE_UAVCAN	0x87
+#define DRV_IMU_DEVTYPE_UAVCAN	0x88
+#define DRV_MAG_DEVTYPE_UAVCAN	0x89
```

Comment:
Some of these (eg DRV_MAG_DEVTYPE_UAVCAN) might correspond to calibrations (device ids).

We should do a big cleanup of this list, maybe even split it by type.

Reply:
I wondered if order preservation mattered and looked back at the last change you made and inferred it did not. If we want to classify them then we should have reserved blocks. But we can fix it.

Reply:
Done